iamchrissmith/uncorked
# adds initial styling with navbar and dynamic page titles

#### What does  this PR do?

- This PR adds materialize CSS
  - Adds a container around the main body
  - Adds styling to the nav bar
  - Adds `mobile_nav.js` for the popout nav for mobile
- Also adds dynamic page titles, to add a dynamic title to new pages add a `<% @title = "Your page title here" %>` 
- Adds an admin user to seed file

#### Where should the reviewer start?

With `_nav.html.erb` partial

#### How should this be manually tested?

The home page will not load right now until the users are built out. Check screenshots.

#### Any background context you want to provide?

None

#### What are the relevant tickets?

none


#### Screenshots (if appropriate)


Standard desktop nav:

<img width="1680" alt="screen shot 2017-07-12 at 3 00 27 pm" src="https://user-images.githubusercontent.com/5731483/28139863-1b687fee-6713-11e7-9cc5-eb6fc423a6f9.png">

Mobile nav before you click the hamburger:

<img width="867" alt="screen shot 2017-07-12 at 3 00 39 pm" src="https://user-images.githubusercontent.com/5731483/28139872-218b31fa-6713-11e7-9e78-4aee5271c50f.png">

Mobile nav after you click the hamburger

<img width="869" alt="screen shot 2017-07-12 at 3 00 47 pm" src="https://user-images.githubusercontent.com/5731483/28139874-24597798-6713-11e7-86e0-9e4fc5f50686.png">

Tab tile:

<img width="190" alt="screen shot 2017-07-12 at 3 03 13 pm" src="https://user-images.githubusercontent.com/5731483/28139924-6107f282-6713-11e7-9bf9-0c92331f3e98.png">


#### Questions:
  - Do Migrations Need to be ran?
  - Do Environment Variables need to be set?
  - Any other deploy steps?
